src/sssom_schema/schema/sssom_schema.yaml

```
@@ -539,8 +539,11 @@ slots:
       - value: 0.95
         description: A confidence score of 0.95, indicating 95% confidence.
   subject_match_field:
-    description: A list of properties (term annotations on the subject) that was used
-      for the match.
+    description:  >
```

```suggestion
    description: >
```
This is just me being nitpicky. The extra space doesn't affect anything, but the inconsistency annoys me. 😄